src/main/java/com/brightsparklabs/dropwizard/bundles/auth/external/ExternallyAuthenticatedAuthFilterFactory.java

```diff
@@ -55,8 +58,8 @@ public abstract class ExternallyAuthenticatedAuthFilterFactory implements Discov
     // -------------------------------------------------------------------------
 
     /**
-     * @return The field in the MDC (Mapped Diagnostic Context) that the authenticated user's
-     *     username is stored against. Default: {@value DEFAULT_MDC_USERNAME_FIELD}.
+     * Returns The field in the MDC (Mapped Diagnostic Context) that the authenticated user's
+     * username is stored against. Default: {@value DEFAULT_MDC_USERNAME_FIELD}.
```

Comment:
You still need the `@return` comment as well, its part of javadoc.
Our practice is to either duplicate the statement. Or allow empty javadopc descriptions in errorprone config, and just use `@return`.